pkg/bosh/manifest/cmd_render_job_tmpls.go

```diff
@@ -113,59 +119,79 @@ func RenderJobTemplates(
 	return nil
 }
 
-func runPreRenderScripts(instanceGroup *InstanceGroup, silent bool) error {
-	for _, job := range instanceGroup.Jobs {
-		for idx, script := range job.Properties.Quarks.PreRenderScripts {
-			createErr := func(err error) error {
-				return errors.Wrapf(err, "failed to run pre-render script %d for job %s for instance group %s", idx, job.Name, instanceGroup.Name)
-			}
+func runRenderScript(jobName string, typeScript string, scripts []string, igName string, silent bool) error {
```

Comment:
I'd appreciate if this parameter list could become multi-line as well.

Reply:
`scriptType` sounds more natural than `typeScript` (which is also a programming language :laughing:).